AAI/FAQ.md

```diff
@@ -459,9 +459,7 @@ of the AAI standard regarding the presence of JKUs.
 
 {% hr2 %}
 
-## Threat Analysis
-
-### What is the danger of using a fully scoped (or audience-less) token in a multi-node workflow
+### What is the limitation of using a fully scoped (or audience-less) token in a multi-node workflow?
```

Comment:
I suggest the heading be changed from "Threat Analysis" to "Known Limitations" and this line be changed to "What are the pros and cons of using a fully scoped (or audience-less) token in a multi-node workflow?"

Reply:
I agree, "Known Limitations" is more accurate heading for this section.

Reply:
I have made the change, thanks!